### DISCUSSION TITLE
Report notification stopped working?

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#3051] [Q&A] [answered]
Yes, I understand that Rockstar doesn’t care about reports, but still. Report protection stopped working?

**Comment by GreySideMM** [comment]
How do you know? I mean, srsly, it's hard to know. I never had this occur but also no one threatened to report me, so idk. 
I'm only aware of REPORT_MYSELF (as cheater; send to everyone in the session) got disabled by R*.

**Comment by ghost** [comment]
> How do you know? I mean, srsly, it's hard to know. I never had this occur but also no one threatened to report me, so idk. I'm only aware of REPORT_MYSELF (as cheater; send to everyone in the session) got disabled by R*.

I know that they sent me a report, but I don’t receive a notification. Perhaps now, just like with chat, everything goes straight to the server.

**Comment by Yimura** [comment, marked as answer]
It’s no longer possible to block reports from other players neither do we receive them on our end.
